build-ps/percona-xtradb-cluster.spec

```diff
@@ -1539,6 +1545,8 @@ fi
 %attr(755, root, root) %{_bindir}/wsrep_sst_mysqldump
 %attr(755, root, root) %{_bindir}/wsrep_sst_xtrabackup-v2
 %attr(755, root, root) %{_bindir}/wsrep_sst_rsync
+%attr(755, root, root) %{_bindir}/ps-admin
```

Comment:
Why ps-admin?
1. If it's going to be shipped with PXC it needs changes there because it includes options to install TokuDB and RocksDB which are not part of PXC - and also other plugins need to be checked.
2. If it will be added to PXC currently I see it added only in `rpm` and not `deb` - why difference?

Reply:
fixed